modules/compliance-applying.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-remediation.adoc
+
+[id="compliance-applying_{context}"]
+= Applying a remediation
+
+The boolean attribute `spec.apply` controls whether the remediation should be applied by the Compliance Operator.  We can apply the remediation by setting the attribute to true:
+
+[source,terminal]
+----
+$ oc patch complianceremediations/sysctl-net-ipv4-conf-all-accept-redirects --patch '{"spec":{"apply":true}}' --type=merge
+----
```

Comment:
`scan_name` should get prefixed to rule like below
```
$ oc patch complianceremediations/worker-scan-sysctl-net-ipv4-conf-all-accept-redirects

$ oc get compliancescan
NAME          PHASE   RESULT
worker-scan   DONE    NON-COMPLIANT

$ oc get complianceremediations/worker-scan-sysctl-net-ipv4-conf-all-accept-redirects
NAME                                                    STATE
worker-scan-sysctl-net-ipv4-conf-all-accept-redirects   NotApplied
```

Reply:
Good catch! I'll change it to `scan-name-systctl-net-ipv4-conf-all-accept-redirects`  and illustrate it in the text above.